### PR TITLE
Allow Bots

### PR DESCRIPTION
**Pull request by jreyes4**
Excludes Bots from MemeNarcs policing

### CODE REVIEW

File: Budgett/config.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
     "prefix": "~",
     "generalChatID": "391063932535701505",
-    "testMode": "false"
+    "testMode": "false",
+    "wednesdayPath": "https://9gag.com/tag/it-is-wednesday-my-dudes",
+    "nGagPath": "https://9gag.com/gag/",
+    "defaultMeme": "https://www.youtube.com/watch?v=OzQ-KvxLVT0",
+    "STGID": "352255685183930369"
```
Comment (PhilipBeckwith) [inline Budgett/config.json:8]: Not a blocker but can we spell this out rather than using aberrations?

File: Budgett/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        .then(responseText => {
+            const doc = (new DomParser).parseFromString(responseText, 'text/html');
+            const arrMemes = doc.getElementById("jsid-latest-entries").textContent.split(',');
+            let rdmMeme = arrMemes[Math.floor(Math.random() * arrMemes.length)];
+            return nGagPath + rdmMeme;
+        })
+        .catch(() => {return defaultMeme})
+}
+
+module.exports = getWednesdayMeme;
```
Comment (PhilipBeckwith) [inline Budgett/utils/utils.js:17]: Do we plan on putting more modules in this file or are we planning to make a new file for each utility we want to build? If we're going to leave it as one utility per file, perhaps we should name this file to be more descriptive.

File: MemeNarc/index.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ client.on('ready', () => {
 
 client.on("message", message => {
     console.log(message);
-    if(message.author.username != userName && message.author.id != id) { // So it doesn't talk to its self
+    if(message.author.bot == false) { //Any bot can post memes
```
Comment (PhilipBeckwith) [inline MemeNarc/index.js:30]: 👍

File: Budgett/utils/utils.js (hunk 2 above)
Comment (jreyes4) [reply Budgett/utils/utils.js:17]: Was planning on adding more modules as we move forward. Not sure what those would be, but I'm not too crazy about adding a new file per feature.

File: Budgett/config.json (hunk 1 above)
Comment (jreyes4) [reply Budgett/config.json:8]: Updated

**Pull request merged**